bocas/artifacts/keras_history.py

```diff
@@ -1,16 +1,29 @@
-import pickle
-
 from bocas.artifacts.artifact import Artifact
+from .yaml_utils import parse_yaml_node
```

Comment:
same

Reply:
Adjusted

Reply:
It was needed, but I agree this was not clean. To make this cleaner, I adjusted line 21. Since `.fit()` returns a history object and not just a dictionary we needed to adjust accordingly for yaml. The `to_yaml()` function now takes on this logic in a nicer way.